include/pipert/Protocol.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PROTOCOLHANDSHAKE_H_
+#define _PROTOCOLHANDSHAKE_H_
+
+#include "pipert/UDPConnection.h"
+
+#include <cstddef>
+
+namespace pipert {
+
+template <class T>
+class Protocol {
+ public:
+  static char HandshakeSuccessMsg[];
```

Comment:
As I see, these could also be static const and according to this, you could call them kHandshakeSuccessMsg...